### PR TITLE
add blog

### PR DESCRIPTION
**Pull request by albanelia0**

### CODE REVIEW

File: staff/albanelia0/blog/index.html [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+    <article>
+      <div>
+        <p>Propellers with mushrooms, tomatoes and butternut squash</p>
+        <img src="images/champiñones.jpg" alt="Propellers with mushrooms" width="250">
+        <h3>Preparation</h3>
+        <p>
+          Lorem ipsum dolor sit, amet consectetur adipisicing elit. Minus ab mollitia quos itaque eveniet! Ratione repellendus maiores,
+          voluptatibus corrupti eum quos, facilis vero perspiciatis architecto, quas dolor accusamus deleniti aliquam.
+        </p>
+      </div>
```
Comment (RominaFior) [inline staff/albanelia0/blog/index.html]: Albita, creo que no es necesario utilizar divs en este ejercicio

File: staff/albanelia0/blog/index.html [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+      <input type="text" placeholder="Write your mail">
+    </nav>
+    
+
+  </footer>
+
+
+  
+</body>
+</html>
```
Comment (RominaFior) [inline staff/albanelia0/blog/index.html]: muy bonito y completo 👍

File: staff/albanelia0/blog/index.html [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+        <li><a href="Tickets">Tickets</a></li>
+        <li><a href="Menu">Menu</a></li>
+      </ul>
+    </nav>
+  </header>
+  <section>
+    <article>
+      <h2>Main courses</h2>
+      <p>Pastas</p>
+      <div>
```
Comment (lgimeneza) [inline staff/albanelia0/blog/index.html]: div is not semantic

File: staff/albanelia0/blog/index.html [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+        <p>Spaghetti carbonara in 15 minutes</p>
+        <img src="images/images.jpeg" alt="Spaghetti carbonara">
+        <h3>Preparation</h3>
```
Comment (lgimeneza) [inline staff/albanelia0/blog/index.html]: maybe in a header?

File: staff/albanelia0/blog/index.html [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+      <h3>contact me</h3>
+      <ul>
+        <li>
+          <a href="mailto:pepito@mail.com)subject=contact from the web&body=Hello!">Contact</a>
+        </li>
+        <li><a href="Telephone">Telephone</a></li>
+      </ul>
+      <h4>Subscription</h4>
+      
+      <input type="text" placeholder="Write your mail">
```
Comment (manuelbarzi) [inline staff/albanelia0/blog/index.html]: WHY is this <input> here? if you wanted to add a form, please, wrap this input inside a form.

SEE https://www.w3schools.com/html/html_forms.asp

File: staff/albanelia0/blog/index.html [hunk 6]
```diff
@@ -0,0 +1,72 @@
...
+          voluptatibus corrupti eum quos, facilis vero perspiciatis architecto, 
+          quas dolor accusamus deleniti aliquam.
+        </p>
+      </div>
+    </article>
+  </section>
+   
+  <section>
+    <article>
+      <div>
```
Comment (manuelbarzi) [inline staff/albanelia0/blog/index.html]: WHY divs? the challenge just only asks for semantics, and div's are not semantic.

File: staff/albanelia0/cv/index.html [hunk 7]
```diff
@@ -27,20 +64,24 @@ <h4>Funciones cómo Auxiliar:</h4>
...
-  <h2>Formación Académica</h2>
-  <h3>CFGM Atención a Personas en Situación de Dependencia</h3>
+  </section>
+  <section>
+    <h2>Formación Académica</h2>
+    <h3>CFGM Atención a Personas en Situación de Dependencia</h3>
+  </section>
+  </main>
 </body>
 </html>
```
Comment (RominaFior) [inline staff/albanelia0/cv/index.html:87]: Muy bien estructurado 👍

File: staff/albanelia0/cv/index.html [hunk 8]
```diff
@@ -7,18 +7,55 @@
   <title>Curriculum</title>
 </head>
 <body>
-  <h1>Curriculum Alba</h1>
-  <h2>Datos personales</h2>
+  <header>
+    <h1>Curriculum Alba</h1>
+  </header>
+  <aside>
```
Comment (manuelbarzi) [inline staff/albanelia0/cv/index.html:13]: WHY ```<aside>``` here? all inner content seems to be ```<main>``` content, isn't it?

File: staff/albanelia0/blog/index.html (hunk 3 above)
Comment (albanelia0) [reply staff/albanelia0/blog/index.html]: 👍 removed div!

File: staff/albanelia0/blog/index.html (hunk 5 above)
Comment (albanelia0) [reply staff/albanelia0/blog/index.html]: Add form! 👍

File: staff/albanelia0/blog/index.html (hunk 6 above)
Comment (albanelia0) [reply staff/albanelia0/blog/index.html]: 👍 removed div!

**Pull request merged**